package.json

```diff
@@ -1,25 +1,26 @@
 {
-  "name": "hetmech-frontend",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
-    "react-scripts": "2.1.3"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
```

Comment:
So you are not going to use `eslint`?

Reply:
I will be eventually. I'll probably end up using multiple and various linting tools for all of the files, not just javascript.